internal/appsec/dyngo/instrumentation/httpsec/actions.go

```diff
@@ -0,0 +1,81 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package httpsec
+
+import (
+	"net/http"
+)
+
+// Action is used to identify any action kind
+type Action interface {
+	isAction()
+}
+
+// BlockRequestAction is the parameter struct used to perform actions of kind ActionBlockRequest
+type BlockRequestAction struct {
+	Action
+	// Status is the return code to use when blocking the request
+	Status int
+	// Template is the payload template to use to write the response (html or json)
+	Template string
+	// handler is the http handler be used to block the request (see wrap())
```

Comment:
```suggestion
	// handler is the http handler to use to block the request (see wrap())
```

Reply:
`wrap()` doesn't exist, right?

Reply:
Not anymore, nice catch. Edited you suggestion to remove that part too.